README.md

```diff
@@ -26,14 +26,21 @@ the reported metrics will be ignored at the backend.
 
 ```java
 import com.lightstep.tracer.metrics.Metrics;
-import com.lightstep.tracer.metrics.SafeMetricsReporter;
+import com.lightstep.tracer.metrics.OkHttpSender;
+import com.lightstep.tracer.metrics.Sender;
 ...
   // Done once, at application initialization.
-  Metrics metrics = SafeMetricsReporter.createMetricsThread(
+  Sender<?,?> sender = new OkHttpSender(
       "MyServiceName", // Service/Component name.
-      "https://" // Metrics url
       "MyAccessToken", // Access Token. Nullable
       "1.2.3", // Service version. Nullable
+      "https://" // Metrics url
+      30000 // connect timeout in milliseconds
+  );
+
+  Metrics metrics = new Metrics(
```

Comment:
Can we mention `Metrics` inherits from `Thread`? This way it makes more sense as to why users should keep a handle to it.

Reply:
// Metrics inherits from Thread.
2 line below

Reply:
Can we move that line here? ;)

Reply:
done